dashboard/config/programming_expressions/applab/accelerometer.json

```diff
@@ -3,7 +3,7 @@
   "name": "accelerometer",
   "category": "Circuit",
   "category_key": "circuit",
-  "content": "The `accelerometer` is an object representing the accelerometer sensor attached to the Circuit Playground.\r\n\r\n### Properties and Methods\r\n\r\n* `accelerometer.getOrientation()` - Get the orientation of the board (i.e tilting forward, back, side to side, spinning, etc)\r\n* `accelerometer.getAcceleration()`- \r\n\r\n### Events\r\n\r\nThe accelerometer has a fair amount of event types as well. \r\n\r\n| Event |      \t\t\t\tDescription         |\r\n|-------|-------------------------------|\r\n| Change |Fired any time the accelerometer experiences a change |\r\n| Data | Fires every 50 milliseconds|\r\n| Single Tap | Fires only when a tap is detected next to the board. |\r\n| Double Tap | Fires only when two taps within x amount of time of each other is detected next to the board. |\r\n",
+  "content": "The [`accelerometer`(#f78183)](/docs/applab/accelerometer/) is an object representing the accelerometer sensor attached to the Circuit Playground.\r\n\r\n### Properties and Methods\r\n\r\n* [`accelerometer.getOrientation()`(#f78183)](/docs/applab/accelerometer.getOrientation/) - Get the orientation of the board (i.e tilting forward, back, side to side, spinning, etc)\r\n* [`accelerometer.getAcceleration()`(#f78183)](/docs/applab/accelerometer.getAcceleration/)- \r\n\r\n### Events\r\n\r\nThe accelerometer has a fair amount of event types as well. \r\n\r\n| Event |      \t\t\t\tDescription         |\r\n|-------|-------------------------------|\r\n| Change |Fired any time the accelerometer experiences a change |\r\n| Data | Fires every 50 milliseconds|\r\n| Single Tap | Fires only when a tap is detected next to the board. |\r\n| Double Tap | Fires only when two taps within x amount of time of each other is detected next to the board. |\r\n",
```

Comment:
`accelerometer` links to itself here, but that's also what happens in CB so I decided not to go back and fix this.